docs/README.md

```diff
@@ -1,26 +1,80 @@
-# Intro to Documentation
+<head>
 
-Documentation involves the creation, organization, and communication of information for the use of developers, players, and the general audience. This documentation takes various forms, from developmental resources to official documentation
+</head>
```

Comment:
what are the `head` tags suppose to do?

Reply:
css?

Reply:
why? it's a markdown file it doesn't need something like this

Reply:
k